pkg/sh/net.go

```diff
@@ -68,30 +69,33 @@ func DownloadS3(ctx *task.Context, from S3Object, toPath string, profile string)
 	return err
 }
 
-// UploadS3 uploads a file to S3.
-func UploadS3(ctx *task.Context, fromPath string, to S3Object, profile string) error {
+// UploadFileToS3 reads the file at the provided path and uploads the contents to S3.
+func UploadFileToS3(ctx *task.Context, fromPath string, to S3Object, profile string) error {
```

Comment:
I think we need to leave the name of this alone. Otherwise, we may break someone.

Reply:
Changed back.